10_install_server/12_install_root_server_rhel.txt

```diff
@@ -0,0 +1,99 @@
+=== Install Rudder Root server on Enterprise Linux ( RedHat or RedHat-like )
+
+==== Add the Rudder packages repository
+
+Run the following command as root:
+
+Note: <version> is your RHEL version
```

Comment:
Of course they *can*. I *can* build my own kernel - but I love that I don't have to.

Also, this documentation is about getting people to test, use and love our software. The easier we make it for them, the less error-prone, the more likely people are to use, adopt, love and contribute to Rudder - which is our ultimate goal, obviously.

So, please bear this in mind. Our job is to ease users lives. This is an easy target: make it simple for them to copy/paste installation instructions. So, please, no variables in the code.

If you can find a command like the one we use in the debian section (that must be installed by default!), then fine. Otherwise, sure, focus on RHEL 7 and add a note to change it for RHEL 6 if necessary. So long as we're not providing extra work for users.